.github/workflows/ci.yml

```diff
@@ -616,7 +616,7 @@ jobs:
           NC="\e[0m"
           set -e
           rc=0
-          changed_files="${{ steps.docs-changed-files.outputs.all_changed_files }}"
+          changed_files="${{ steps.docs-changed-files.outputs.added_files }} ${{ steps.docs-changed-files.outputs.modified_files }}"
```

Comment:
This one look good overall, but we'll need to modify `if [ -n "$changed_files" ]; then` to account for `$changed_manifest` - otherwise (or-condition) manifest will not be validated if md-files are removed and renamed.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/56081/commits/0f666ae5d20f42adab9db2904ff0051b51be7ecc